docs/user-guide/configuring.md

```diff
@@ -563,6 +563,23 @@ In this example, the `eslint-config-myrules` package will be loaded as an object
 
 **Note:** You can omit `eslint-config-` and ESLint will automatically insert it for you, similar to how plugins work. See [Shareable Configs](../developer-guide/shareable-configs) for more information.
 
+ESLint also supports extending configuration from plugins that provide configs:
+
+```js
+{
+    "extends": "plugin:eslint-plugin-myplugin/default",
+    
+    "rules": {
+        // Override any settings from the "parent" configuration
+        "eqeqeq": 1
+    }
+}
+```
+
+In this example, the `eslint-plugin-myplugin` package contains configuration named `default`.
+
+**Note:** Please not that when you are extending from configuration bundled with plugins, you need to start with `plugin:` prefix as well as specify configuration name after the slash. You can also omit `eslint-plugin-` part.
```

Comment:
The last sentence needs "the". Would "You may optionally omit the `eslitn-plugin-` prefix." be clearer?